spec/APNS.spec.js

```diff
@@ -1,68 +1,31 @@
-var APNS = require('../src/APNS');
+let APNS = require('../src/APNS');
 
 describe('APNS', () => {
 
-  it('can initialize with single cert', (done) => {
-    var args = {
-      cert: 'prodCert.pem',
-      key: 'prodKey.pem',
+  it('can initialize with cert', (done) => {
+    let args = {
+      cert: new Buffer('testCert'),
```

Comment:
it should work, if the string contains a (resolvable) filename or a string, that matches against `/-----BEGIN ([A-Z\s*]+)-----/`. See [here](https://github.com/node-apn/node-apn/blob/master/lib/credentials/resolve.js#L9) for details.
Will change one of the test-cases to test this.